### DISCUSSION TITLE
MOES Tuya TRV ZigBee3 Valve

### DISCUSSION THREAD

**Discussion by Vodny86** [Koenkk/zigbee2mqtt#11071] [Devices]
Good day.

I would like to ask one thing here. 
Can I connect these (https://www.aliexpress.com/item/4001065119510.html?spm=a2g0o.9042311.0.0.54b54c4dMCkr4F) MOES valves to HA (RPi4B)? I think so, but HUB probably won't use it, will it? I didn't find it on supported devices :-( I have wired(eth) model

If the hub will not be applicable, is a good alternative this?[ Https: //www.aliexpress.com/item/1005003763454756.html? Spm = a2g0s.8937460.0.0.10dc2e0etAtkXG](https://www.aliexpress.com/item/1005003606767695.html?spm=a2g0o.productlist.0.0.34ab78960HvPiQ&algo_pvid=6553bbf8-d041-41b9-9fbc-140d072e8e27&algo_exp_id=6553bbf8-d041-41b9-9fbc-140d072e8e27-0&pdp_ext_f=%7B%22sku_id%22%3A%2212000026929305754%22%7D&pdp_pi=-1%3B16.94%3B-1%3B644%40salePrice%3BEUR%3Bsearch-mainSearch)

Thank you very much

**Comment by groenmarsmannetje** [comment]
Yes, it probably will be recognized as this device https://www.zigbee2mqtt.io/devices/TS0601_thermostat.html#tuya-ts0601_thermostat

There are different manufacturers with different version numbers available that all look the same.
